src/backend/templates/schedule.hbs

```diff
@@ -539,9 +542,18 @@
                     $(sessionName).html(textArray.join(' ') + '...');
                 }
               })
+              roomCount++;
             });
+
             if(width < $(rooms).width()) {
-              $(rooms).width(width);
+              offset = ($(rooms).width() - width)/roomCount;
```

Comment:
@aayusharora Yes I agree with you

- We will need to store the width on the server, and we will stop using `min-width` & 'max-width' in css.
- We will calculate and set the width in handlebar while generating HTML.